remoulade/actor.py

```diff
@@ -239,7 +242,22 @@ def send_with_options(self, *, args=None, kwargs=None, delay=None, **options) ->
         return self.broker.enqueue(message, delay=delay)
 
     def as_dict(self) -> ActorDict:
```

Comment:
this function need to be well tested to make sure it works for all cases: with/without default, with/without typing/ with/without, using `*args` and `**kwargs`, ...